kubeflow/instance/Kptfile

```diff
@@ -37,6 +37,11 @@ openAPI:
         setter:
           name: location
           value: LOCATION
+    io.k8s.cli.setters.zone:
+      x-k8s-cli:
+        setter:
+          name: zone
```

Comment:
There seems to be a mismatch, zone parameter should be set by
```
kpt cfg set ./upstream/manifests/gcp gcloud.compute.zone $(ZONE)
```

So maybe it should look like:
```
io.k8s.cli.setters.gcloud.compute.zone:
```
?

I also have a concern that we will eventually default to regional clusters, can we change the parameter to location, but document currently we only support zone here.

Reply:
I think we should potentially have setters for both location and gcloud.compute.zone. 

Which setter gets used in a particular location should then depend on whether the value must be zonal  (use gcloud.compute.zone) or can be regional or zonal (use location).

Most of the current usages can use either (location) e.g. cluster location.

I think one place where we have a zonal resource is the PD backing the KFP MySQL.

Reply:
When we move to a regional compatible setup, IMO there shouldn't be any zonal resources left. (e.g. we should replace those PDs with gcs and regional/zonal cloudsql)
But we can leave this to a later topic.